README.md

```diff
@@ -31,8 +32,8 @@ compilers: [:thrift | Mix.compilers]
 ```
 
 It's important to add `:thrift` *before* the `:elixir` entry. The Thrift
-compiler will generate Elixir source files, and we currently rely on this
-ordering to ensure those generated source files get compiled.
+compiler will generate Elixir source files, which will then be compiled by the `:elixir` compiler.
+We currently rely on this ordering to ensure those generated source files get compiled.
```

Comment:
Suggestion: ", which are in turn compiled during the normal `:elixir` compile step." and strike the final sentence.